src/remove_all_completed.js

```diff
@@ -0,0 +1,9 @@
+import storage from './store.js';
+
+const clearAllCompleted = () => {
+  let list = storage.get();
+  list = list.filter((task) => !task.completed);
+  storage.setList(list);
+};
```

Comment:
- [ ] `clearAllCompleted` method doesn't update indexes of the items after removing completed tasks from the list.
And if I try to add a new item to the list afterward it causes duplicate indexes.
![Screencast from 08-29-2021 12_22_08 AM](https://user-images.githubusercontent.com/1642014/131231371-8e38438a-6003-4a57-bc8d-5ff34526eb64.gif)

Reply:
:+1: This issue is resolved :heavy_check_mark: